src/editables/__init__.py

```diff
@@ -53,18 +29,21 @@ def build_editable(location, expose=None, hide=None):
     location = Path(location)
 
     if expose:
-        for e in expose:
-            if not (location / e / "__init__.py").is_file():
-                raise ValueError("{} is not a package in {}".format(e, location))
+        for pkg in expose:
+            if not (location / pkg / "__init__.py").is_file():
+                raise ValueError("{} is not a package in {}".format(pkg, location))
     else:
         expose = [pkg.parent.name for pkg in location.glob("*/__init__.py")]
 
-    if hide:
-        for h in hide:
-            if not any(h.startswith(e) for e in expose):
-                raise ValueError("{} is not part of an exposed package".format(h))
```

Comment:
I don't think we should be in the business of validating the inputs. Let the caller take care of that; e.g. there's a reason to put something into excluding even if it does not exist yet: the user might create it later and we want to not need to reinstall in this case.

Reply:
Looks like your comments got out of order. See my reply above. I see your point here, but I'd prefer to start by validating, and review once we have a good use case that matches the situation you describe.

Reply:
I just described a use case above.

Reply:
... and I've come to the conclusion that you were right about this one too, at least for now :-)

Reply:
👍 the way I see it the backend is in control of validation (and they likely already have logic in-place to do this); this tool just makes modules designated by the backend for the frontend 🤷

Reply:
I do feel that there are cases we probably should check for, and can do better than the caller, but let's hold off on that until we have a better idea of how tools will use this. It's easier to be lenient now and tighten up later if needed.

Reply:
Totally agree.